src/main/java/org/prebid/server/bidder/loyal/proto/LoyalImpExtBidder.java

```diff
@@ -0,0 +1,18 @@
+package org.prebid.server.bidder.loyal.proto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder
+@Value
+public class LoyalImpExtBidder {
```

Comment:
Confusing name, I suggest using the `LoyalImpExt`

Reply:
I have deleted:
adapters:
  loyal:
    enabled: false

but now module functional tests have filed.

Reply:
remove only `enabled: false`